appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.service.js

```diff
@@ -0,0 +1,16 @@
+/** @typedef {import('../../appeal-details.types.js').WebAppeal} Appeal */
```

Comment:
Again just a tip, I'm fairly sure you can use aliases for these (in this case I think it's `#appeals`), to avoid the relative path